tests/core/utils/test_convert.py

```diff
@@ -38,11 +38,12 @@ def test_subdataset(self):
         with TemporaryDirectory() as directory:
             with utils.ignore_not_georeferenced():
                 paths, names = cogify_subdatasets(infile, directory)
+            names = [name.lstrip("_") for name in names]
             self.assertEqual(
-                names,
+                sorted(names),
                 [
-                    "__MonthlyRainTotal_GeoGrid_Data_Fields_RrLandRain",
-                    "__MonthlyRainTotal_GeoGrid_Data_Fields_TbOceanRain",
+                    "MonthlyRainTotal_GeoGrid_Data_Fields_RrLandRain",
+                    "MonthlyRainTotal_GeoGrid_Data_Fields_TbOceanRain",
```

Comment:
Per https://github.com/stac-utils/stactools/pull/396#issuecomment-1431713557 we can drop this fix, CI should let the original code pass now.

Reply:
I removed my changes.